### ISSUE TITLE
47077422: Hidden Collection View Items Still Visible To VoiceOver

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#20891]
#### Description

Area:
Accessibility

Summary:
When using setting the `hidden` property on an NSCollectionViewLayoutAttributes object, the corresponding item does not occur in the view hierarchy, nor in the accessibility hierarchy (as shown by accessibility inspector). However, when using VO+Left Arrow or VO+Right Arrow to navigate the collection view's accessibility children, hidden items are still read to the user.

Steps to Reproduce:
1. Hide an NSCollectionViewItem using the `hidden` property of NSCollectionViewLayoutAttributes.
2. Navigate the collection view using VoiceOver.

Expected Results:
The item should be invisible to VoiceOver.

Actual Results:
The item is visible to VoiceOver

Version/Build:
macOS Version 10.14.2 (18C54)

-
Product Version: 10.14.2 (18C54)
Created: 2019-01-07T07:33:08.142752
Originated: 2019-01-07T00:00:00
Open Radar Link: http://www.openradar.me/47077422
